app/graphql/mutations/create_provider_and_request.rb

```diff
@@ -41,8 +41,7 @@ def resolve(first_name:, last_name: "", neighborhood: "", city:, state:, email:,
       provider.save!
       request.save!
       linked_token = LinkCreator.create_token(request)
-      # TODO: Do we plan on using Action Job? If so, make async
-      ProviderMailer.with(linked_token: linked_token).request_created_email.deliver_now
+      ProviderMailer.with(linked_token: linked_token).request_created_email.deliver_later
```

Comment:
@tuntisz Added an in-process background worker for now. It's not a long term solution— we should move to sidekiq eventually.